server/app/index.js

```diff
@@ -13,6 +14,7 @@ app.use(morgan('dev'));
 app.use(helmet());
 app.use(compression());
 app.use(sessionMiddleware);
+app.use(sessiodIdMiddleware)
```

Comment:
is this supposed to be spelled "sessionIdMiddleware" instead of "sessiodIdMiddleware"?